service/src/main/java/uk/gov/hmcts/reform/fpl/service/ConfidentialDetailsService.java

```diff
@@ -74,14 +69,9 @@
             all.forEach(element -> {
                 T party = element.getValue();
                 if (party.containsConfidentialDetails()) {
-                    T partyToAdd = getItemToAdd(confidential, element);
```

Comment:
Why have we renamed this method? It isn't adding confidentialDetails, it is getting the confidential item to add

Reply:
yes, you are right. I will call it getConfidentialItemToAdd..